app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @post = Post.find(params[:post_id].to_i)
```

Comment:
Think of how to nest your resources: http://guides.rubyonrails.org/routing.html#nested-resources